server/sonar-server/src/main/java/org/sonar/server/permission/UserPermissionChanger.java

```diff
@@ -54,7 +48,12 @@ public boolean apply(DbSession dbSession, UserPermissionChange change) {
         break;
       case REMOVE:
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce this switch case number of lines from 8 to at most 5, for example by extracting code into methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1151)